linter_exclusions.yml

```diff
@@ -13,3 +13,10 @@ dt endpoint create eventhub:
     eventhub_resource_group:
       rule_exclusions:
       - parameter_should_not_end_in_resource_group
+central app cde update:
```

Comment:
You will need to add the "iot" prefix for the command group. In the digital twin case, dt is the root group.